pkgs/applications/misc/mepo/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, stdenv, fetchFromSourcehut, pkg-config, zig
+, curl, SDL2, SDL2_image, SDL2_ttf
+}:
+
+stdenv.mkDerivation rec {
+  pname = "mepo";
+  version = "0.2";
+
+  src = fetchFromSourcehut {
+    owner = "~mil";
+    repo = pname;
+    rev = version;
+    hash = "sha256-ECq748GpjOjvchzAWlGA7H7HBvKNxY9d43+PTOWopiM=";
+  };
+
+  nativeBuildInputs = [ pkg-config zig ];
+
+  buildInputs = [ curl SDL2 SDL2_image SDL2_ttf ];
+
+  buildPhase = ''
+    export HOME=$TMPDIR
+    zig build -Drelease-safe=true
```

Comment:
Maybe we should have `-Dcpu=baseline` here.

BTW, I don't fully understand why, but other Zig packages have HOME set in preBuild and `zig build` with `--prefix $out` in installPhase.  Probably does not matter.

Reply:
> Maybe we should have `-Dcpu=baseline` here.

Agree, that's make sense.

> BTW, I don't fully understand why, but other Zig packages have HOME set in preBuild and `zig build` with `--prefix $out` in installPhase. Probably does not matter.

Yes, it's actually the same result. Other packages just build in `installPhase` instead of `buildPhase`.